.github/workflows/lza_dev_deployment.yml

```diff
@@ -0,0 +1,46 @@
+name: LZA DEPLOY Development (DEV) Environment
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "main"
+
+concurrency:
+    group: deploy-dev-global # Ensure only one of this job can be ran at a given time
+
```

Comment:
I just realized we didn't include the permission section in our tools and dev deployment file, maybe we should add it?

Reply:
Hi Catherine, the reusable yml already has the permission, so no need to add it here..